### PR TITLE
Fix post ranking docs in about guide. Fixes #1294

### LINKED ISSUE #1294: Forum mode \ sort by recently commented
I think it would be really useful to have a Forum mode sorting option. Especially for smaller\slower communities it would be great to see topics go to the top based on which has been recently commented on and have any thread I comment on give me a notification if there's been new replies whether or not its been to my comment.

### PR DESCRIPTION
**Pull request by dessalines**

### CODE REVIEW

File: docs/src/about_guide.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ Start typing...
 
 Type | Description
 --- | ---
-Hot | Shows *trending* posts, based on the score, and the most recent comment time.
+Active | Shows *trending* posts, based on the score, and the most recent comment time.
+Hot | Shows *trending* posts, based on the score, and the post creation time.
```
Comment (Nutomic) [inline docs/src/about_guide.md:16]: Ah so the main difference is that one is based on comment activity, and the other on post activity. Its hard to figure out from these sentences that are almost identical. How about:

> Trending sort based on votes and comment activity

Cause this is supposed to be a simple explanation for new users. Detailed explanation is in another document (and that could go more in depth about the details of each sorting method.

Comment (dessalines) [reply docs/src/about_guide.md:16]: most recent comment time is simple enough.

Comment (Nutomic) [reply docs/src/about_guide.md:16]: Maybe just put that part first?

Comment (dessalines) [reply docs/src/about_guide.md]: I updated, is that one okay?

Comment (Nutomic) [reply docs/src/about_guide.md]: Yes, sounds better to me.

**Pull request merged**